tools/postgres/ensure_schema.sh

```diff
@@ -8,7 +8,8 @@ dbHost="$("$scriptdir"/db_host.sh)"
 
 schemaPath="$scriptdir/schema.sql"
 
-if [ "$(PGPASSWORD=postgres psql -U postgres -h  $dbHost --dbname=$dbName -tAc "SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'webknossos';")" = 'webknossos' ]
+SCHEMA_EXISTENCE="$(PGPASSWORD=postgres psql -U postgres -h  $dbHost --dbname=$dbName -tAc "SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'webknossos';")"
```

Comment:
How about schemaName ? the variable does not really contain the existence, right? Also, the other vars are not all-caps.

Reply:
```suggestion
schemaName="$(PGPASSWORD=postgres psql -U postgres -h  $dbHost --dbname=$dbName -tAc "SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'webknossos';")"
```